src/main/java/org/fundacionjala/pivotal/pages/project/PageLogin.java

```diff
@@ -32,6 +45,11 @@ private void setTxtUsername(final String username) {
         txtUsername.sendKeys(username);
     }
 
+    private WebElement getLogoPivotal() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "getLogoPivotal" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUnusedPrivateMethod)